### DISCUSSION TITLE
Readd Chrome's "Disable hardware acceleration".

### DISCUSSION THREAD

**Discussion by BNortonnul** [minbrowser/min#1906] [Ideas]
I'm one of a couple of people who has an issue on YouTube, where changing the state of the full-screen-ness causes the screen to briefly flicker black. The solution I found was to disable hardware acceleration on Chrome. However, I was unable to find the equivalent on Min, and switched back to my displeasure. I am very interested in returning to Min, but I'd like this reimplemented or for another workaround for this particular issue.

**Comment by mpgithubcode** [comment]
Yea, would love to be able to edit the chrome://flags but it seems they have them disabled

**Comment by morellicai** [comment]
I know it's not quite the discussion here, but I see that it fits. I use hyprland, and I really needed to be able to change settings in the chrome://flags to be able to change the browser's flag to recognize that it is in wayland so as not to have visual bugs like this in the image I attached here. 
<img width="3840" height="1080" alt="screenshot" src="https://github.com/user-attachments/assets/32fcaddb-e6a7-47eb-9aff-33b9ff76bc2c" />
In the screenshot I took of my two screens, you can see that a window opens on my other screen when I press the 3 dots that are in the left corner of the browser. Without being able to configure flag options, I can't remove this "bug" that is the incompatibility with applications in wayland

**Comment by ioc2e3** [comment]
> I'm one of a couple of people who has an issue on YouTube, where changing the state of the full-screen-ness causes the screen to briefly flicker black. The solution I found was to disable hardware acceleration on Chrome. However, I was unable to find the equivalent on Min, and switched back to my displeasure. I am very interested in returning to Min, but I'd like this reimplemented or for another workaround for this particular issue.

Agree ...

Please add ( Disable hardware acceleration ) option ...

.
